apps/meteor/tests/end-to-end/api/livechat/12-priorites.ts

```diff
@@ -48,9 +49,42 @@ import { IS_EE } from '../../../e2e/config/constants';
 			expect(current).to.have.property('dueTimeInMinutes', sla.dueTimeInMinutes);
 			await deleteSLA(sla._id);
 		});
+		// POST
+		it('should return an "unauthorized error" when the user does not have the necessary permission', async () => {
```

Comment:
Can you separate these on different describes? one for [GET] livechat/sla and the other for [POST]